CMakeLists.txt

```diff
@@ -103,6 +103,15 @@ OPTION(AVX2 "Enable use of the Advanced Vector Extensions 2 (AVX2) instruction s
 
 OPTION(BUILD_TESTING "Enable the SixTest suite" OFF)
 
+OPTION(32BITM "Floats are 32bit (single precision)" OFF)
+OPTION(64BITM "Floats are 64bit (double precision)" ON)
+OPTION(128BITM "Floats are 128bit (quad precision)" OFF)
+
+OPTION(ROUND_NEAR "Always round to the nearest number after floating point operations" ON)
```

Comment:
These should depend on CRLIBM, otherwise they make no sense.

Reply:
Todo in the future :) need an updated CRLIBM first!

Reply:
Not really! It should all be there.